velox/functions/lib/aggregates/SingleValueAccumulator.h

```diff
@@ -43,12 +43,11 @@ struct SingleValueAccumulator {
       vector_size_t index,
       CompareFlags compareFlags) const;
 
-  /// Returns 0 if stored and new values are equal; <0 if stored value is less
-  /// then new value; >0 if stored value is greater than new value.
-  ///
-  /// The caller needs to ensure that hasValue() is true before calling this
-  /// method.
-  int32_t compare(const DecodedVector& decoded, vector_size_t index) const;
+  static int32_t compare(
```

Comment:
Curious, what's the motivation for this change?

Reply:
This doesn't seem right, I'll fix it. I intended to move the `MinMaxAggregate::compare` to SingleValueAccmulator as a static method since it's used in multiple places. I shouldn't remove the original compare method in SingleValueAccmulator, I'll recover it.

Reply:
> I shouldn't remove the original compare method in SingleValueAccmulator, I'll recover it.

@mbasmanova Sorry that I misleading you here. The original compare API needs to be removed as we discussed in  https://github.com/facebookincubator/velox/pull/6548#discussion_r1325937508 .
Both the Presto and Spark min_by/max_by need to use the new API added in https://github.com/facebookincubator/velox/pull/6570.